x-pack/legacy/plugins/uptime/server/lib/adapters/pings/memory_pings_adapter.ts

```diff
@@ -34,10 +34,17 @@ export class MemoryPingsAdapter implements UMPingsAdapter {
       pings = pings.filter(ping => ping.monitor && ping.monitor.id === monitorId);
     }
 
+    const locations =
+      this.pingsDB
+        .map(ping => {
+          return get<string>(ping, 'observer.geo.name');
+        })
+        .filter(ping => !ping) || [];
```

Comment:
This isn't a big deal but the name `ping` isn't very descriptive of what the value of this symbol represents. It's only the value of `observer.geo.name` from the `ping` in the map. A better name would be `location`. Not a huge deal or worth holding up the PR for though.

Reply:
You are 100% right, will change it